quickstart/public/assets/css/common.css

```diff
@@ -60,7 +60,7 @@ body {
     padding-bottom: 2rem;
 }
 
-hanko-auth, hanko-profile {
+hanko-auth, hanko-profile, hanko-logout {
```

Comment:
`hanko-logout` can be removed:

```suggestion
hanko-auth, hanko-profile {
```

Reply:
@bjoern-m done.